### ISSUE TITLE
Request backup plugin

### ISSUE THREAD

**Issue by CtrlAltCris** [Codiad/Codiad#680]
Hey guys, i try found backup system to files like wordpress have.

In fact, i think is nice idea, backup per file (we can define how munch versions per file) when saved.

I dunno how can do this so if someone know, please, share with me :+1: 

Tnks!

**Comment by MSandro** [comment]
Hi, I know this "issue" is closed and was posted in 2014, but it would be realy nice to have have an stand-alone solution without the need of using git or svn. ICEcoder (an other web-ide) creates an snapshot every time the user saves the file. This is the only thing I realy miss in Codiad.

**Comment by Carlosru** [comment]
I have wrote a plugin that creates zip backups , tested only on Linux, the backups is not per file else per all the whole project, take a look at https://github.com/Carlosru/codiadBackupPlugin

**Comment by HLSiira** [comment]
@Carlosru Does this create a backup on every file change or only once per day?

**Comment by Carlosru** [comment]
This will create a zip file of the whole project, i mean a backup of all the files every time you run the option on the right side menu.

Greetings

05/09/2020 escribió:
>@Carlosru Does this create a backup on every file change or only once
>per day?

-- 
Enviado desde mi teléfono con <a href="https://play.google.com/store/apps/details?id=com.onegravity.k10.pro2"><b>K-@ Mail</b></a>.

**Comment by HLSiira** [comment]
That's pretty cool. Would you mind me porting it to my updated version of Codiad? Unless you want to do the porting.
